.github/scripts/wave_singularity.py

```diff
@@ -8,47 +8,79 @@
 
 import logging
 
-import httpx
+import requests
+import rich_click as click
+from rich.logging import RichHandler
```

Comment:
maybe we should add a requirements.txt or similar to make install these dependencies?

Reply:
Depends if we expect anyone except GitHub Actions runner to run this. If we do, then we should probably stick it in nf-core/tools in fairness..

If it's just for the GHA runner I don't really care much

Reply:
I was just thinking it might be easier to keep track of things if one needs to add them to a requirements.txt compared to a step in an action, but maybe in the end its the same amount of work

Reply:
pfff, reading docs. totally overrated.